src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link, NavLink, withRouter } from 'react-router-dom'
+
+const Navbar = (props) => {
+  return (
+    <nav>
+      <div className="Navbar">
```

Comment:
we can just put this "Navbar" class directly to the nav tag

Reply:
I moved className from dev to nav. thanks for the advice!